velox/docs/monthly-updates/december-2021.rst

```diff
@@ -0,0 +1,38 @@
+********************
+December 2021 Update
+********************
+
+Documentation
+-------------
+
+* Add :doc:`Aggregation-related optimizations <../develop/aggregations>` guide.
+* Add missing documentation for :func:`json_extract_scalar` function.
+* Add documentation for missing functions in :doc:`Mathematical Functions <../functions/math>` and :doc:`Comparison Functions <../functions/comparison>`.
+* Update :doc:`How to add an aggregate function? <../develop/aggregate-functions>` guide to reflect changes to the function registry.
+* Update :doc:`How to add a scalar function? <../develop/scalar-functions>` guide to describe how to write functions with complex types inputs and results.
+
+
+Core Library
+------------
+
+* Add support for partition key aliases to Hive Connector.
+* Add support for conversions between Velox string and Arrow string.
+* Add support for 'timestamp with timezone' in :func:`to_unixtime` function.
+* Add support for :ref:`StreamingAggregation <AggregationNode and StreamingAggregationNode>` operator.
+* Fix a bug in HashJoin filter to enable TPC-H query 19.
+
+Presto Functions
+----------------
+
+* Add :func:`corr`, :func:`covar_samp`, :func:`covar_pop`, :func:`every` aggregate functions.
+* Add :func:`sort_array`, :func:`array_sort`, :func:`array_position` array functions.
```

Comment:
sort_array is not a Presto function. Would you double check?

Reply:
I wonder if this is from the following commit:

```
46db7f9	Naresh Kumar	Wed Dec 8 18:38:05 2021 -0800	Enable support for sort_array and array_sort functions (#707)
```

If so, this commit applies to Spark functions, not Presto functions.

```
% git show 46db7f9 --name-only
commit 46db7f9c7cf8bcc25d166396eb09a071f27e2686
Author: Naresh Kumar <knaresh@fb.com>
Date:   Wed Dec 8 18:38:05 2021 -0800

    Enable support for sort_array and array_sort functions (#707)

    Summary:
    Pull Request resolved: https://github.com/facebookincubator/velox/pull/707

    Modified Existing ArraySort class to support all array sort variants
    array_sort(array) -> always ascending, nulls are largest
    sort_array(array, isAsc) -> optional ascending flag, nulls are smallest
    fb_array_sort(array, isDesc) -> optional descending flag, nulls are smallest

    Limitations:
    Does not support BOOLEAN and Complex types

    Reviewed By: funrollloops

velox/functions/sparksql/ArraySort.cpp
velox/functions/sparksql/ArraySort.h
velox/functions/sparksql/Register.cpp
velox/functions/sparksql/tests/ArraySortTest.cpp
velox/functions/sparksql/tests/CMakeLists.txt
velox/functions/sparksql/tests/SortArrayTest.cpp
```

CC: @funrollloops @nemausus

Reply:
You are right that these are Spark functions. I added a section for Spark Functions. Let me know your thoughts. Thanks.